src/amo/components/AddonReviewList.js

```diff
@@ -55,118 +106,101 @@ export class AddonReviewListBase extends React.Component {
     return `${this.addonURL()}reviews/`;
   }
 
-  renderReview(review: UserReviewType) {
+  renderReview({ review, key }: RenderReviewParams) {
     const { i18n } = this.props;
-    const timestamp = i18n.moment(review.created).fromNow();
+
+    let byLine;
```

Comment:
[Byline](https://www.google.com/search?q=define%3A+byline&ie=utf-8&oe=utf-8) has been a standard word since the 1950s. Naming it `reviewAuthor` is awkward because the code already uses a variable named `review.userName`. Plus, this text contains name and date which makes it more of a byline than just an author name.